README.md

```diff
@@ -179,6 +179,20 @@ jobs:
 See [generate-matrix/action.yml](./generate-matrix/action.yml) for more details.
 This has been available since version 4 of this action.
 
+## Release
+
+To trigger a release (and update the `@v{major}` tag), merge a commit to `main`
+that follows [Conventional Commits][]. In short,
+
+- `fix:` to trigger a patch release,
+- `feat:` for minor, and
+- `feat!:` and major
+
+We don't enforce conventional commits generally (though you are free do so),
+it's only required if you want to trigger release.
```

Comment:
is it now the _only_ way to trigger a release or can someone still do whatever the old method was

Reply:
This is kind of a letter of the law / spirit of the law questions.

Anyone can always do manual things, nothing can really be set up to _prevent_ that. But the steps aren't super intuitive and are made a bit more error prone (you might manually try to update the `v{major}` _branch_, for example, while this automation expects to maintain a `v{major}` _tag_). So I would not _recommend_ folks do manual things.